ballerina-shell/modules/shell-cli/src/main/resources/command.help.txt

```diff
@@ -1,4 +1,4 @@
-Ballerina Shell Help
+Ballerina Shell [experimental] Help
```

Comment:
```suggestion
Ballerina Shell Help
Note: This is an experimental feature, which supports only a limited set of functionalities.
```
WDYT? We are explicitly saying it supports only a limited set of functionalities.

Reply:
We used to have similar message in openAPI and graphQL
- https://github.com/ballerina-platform/ballerina-lang/pull/36299/files#diff-b836d10652985678b3b08494257991011db3219fe77207bfe14c1dcf3bd07863L21-L23
- https://github.com/ballerina-platform/ballerina-lang/pull/36295/files#diff-9cc742e94958fe0aff9397bdb85279260669a76aae0b515a54c3d21b74416c5cL16-L17

Reply:
+1